pages.hi/windows/cd.md

```diff
@@ -0,0 +1,24 @@
+# cd
+
+> वर्तमान निर्देशिका के बारे में जानकारी प्रदान करता है और आपको दूसरे में जाने देता है।
+> अधिक जानकारी: <https://learn.microsoft.com/windows-server/administration/windows-commands/cd>.
+
+- वर्तमान निर्देशिका का मार्ग दिखाएँ:
+
+`cd`
+
+- वर्तमान ड्राइव की मूल तक जाएँ:
+
+`cd \`
+
+- ऊपर की ओर जाएँ:
+
+`cd ..`
+
+- समान डिस्क में एक विशिष्ट निर्देशिका में जाएँ:
+
+`cd {{निर्देशिका\का\मार्ग}}`
+
+- दूसरे डिस्क में एक विशिष्ट निर्देशिका में जाएँ:
+
+`cd /d {{C}}:{{निर्देशिका\का\मार्ग}}`
```

Comment:
```suggestion
`cd {{निर्देशिका\का\पथ}}`

- दूसरे डिस्क में एक विशिष्ट निर्देशिका में जाएँ:

`cd /d {{C}}:{{निर्देशिका\का\पथ}}`
```

`मार्ग` (route) would work fine too here, but we use the transliteration of the word instead [across Hindi pages](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) to not confuse path with route (especially in networking pages).